### DISCUSSION TITLE
:tea: Consider deliver on tea.xyz :nerd_face:

### DISCUSSION THREAD

**Discussion by adriens** [schemacrawler/SchemaCrawler#1044] [Ideas]
That would be a crazy option for oneliners

https://tea.xyz/

**Comment by sualeh** [comment]
@adriens Very interesting. I will have to wait until someone packages a Java CLI application, so I can use their "package.yml" as a template.

**Reply by adriens** [reply]
Yes, I'm in the same case. You can also rely on `brew` as input :bulb:
